application/F3DConfigFileTools.cxx

```diff
@@ -85,7 +85,8 @@ std::vector<fs::path> GetConfigPaths(const std::string& configSearch)
 }
 
 //----------------------------------------------------------------------------
-std::pair<F3DOptionsTools::OptionsEntries, F3DConfigFileTools::BindingsEntries>
+std::tuple<F3DOptionsTools::OptionsEntries, F3DOptionsTools::OptionsEntries,
```

Comment:
It's not clear at first why we do have two options entries. It's probably a good idea to create a struct to name the two lists.